src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {memo} from 'react';
+import {block} from '../utils/cn';
+import './Notifications.scss';
+import {NotificationsEmptyState} from './NotificationsEmptyState';
+import {NotificationsList} from './NotificationsList';
+import {NotificationsProps} from './definitions';
+import i18n from './i18n';
+
+const b = block('notifications');
+
+export const Notifications: React.FC<NotificationsProps> = memo(function Notifications(props) {
```

Comment:
Removed `React.FC`
I think `NotificationContainer` is little bit misleading — it sounds like an empty container to put some notifications in, although in reality it has notifications in it
Additionaly, styling the container is up the the end-user, because one might want to use it in a popup, while another will use it fullscreen (on a touch device for example)